portfolio/src/main/webapp/index.html

```diff
@@ -47,7 +48,11 @@ <h2>Your comments</h2>
       </div>
       <div class="maps-container">
         <h2>Where should I travel and work out of for some time? Check out my options below!</h2>
-        <div id="map"></div>
+        <div id="map" class="chart"></div>
```

Comment:
UX nit: how would you feel about moving this happiness chart to a separate tab within your site to help split up the home page?